src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,59 @@
+package lotto.model;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto() {
+        this(generateRandomNumbers());
+    }
+
+    public Lotto(int number1, int number2, int number3, int number4, int number5, int number6) {
```

Comment:
```suggestion
    public Lotto(int... numbers) {
```
이와 같이 가변 인자로 구현 가능함.
이번 기회에 가변 인자에 대해 추가 학습해 보고 적극적으로 활용해 볼 것을 추천